open-sphere-plugins/feature-actions/src/main/java/io/opensphere/featureactions/controller/StyleFactory.java

```diff
@@ -55,7 +46,7 @@
     public StyleFactory(Toolbox toolbox)
     {
         myToolbox = toolbox;
-        myLabelApplier = new LabelApplier();
+//        myLabelApplier = new LabelApplier();

```

Comment:
I believe we need to keep this and I don't think this is the cause of the ticket.